docs/api/address/get-address-transactions-with-transfers.example.json

```diff
@@ -67,6 +67,22 @@
           "sender": "SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE",
           "recipient": "SP26066SDPP4NXKGCVYZQDR5GX2QPE8KADZ0YK2J7"
         }
+      ],
+      "ft_transfers": [
+        {
+          "amount": "10.53",
+          "asset_identifier": "COOL",
```

Comment:
The actual responses are fully qualified right, should we use a more accurate example? E.g. `SP466FNC0P7JWTNM2R9T199QRZN1MYEDTAR0KP27.miamicoin-token::miamicoin`